tests/CombinedReceive.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# 
+# LSST Data Management System
+#
+# Copyright 2008-2014  AURA/LSST.
+# 
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the LSST License Statement and 
+# the GNU General Public License along with this program.  If not, 
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import unittest
+
+import os
+import platform
+import time
+import lsst.ctrl.events as events
+from lsst.daf.base import PropertySet
+
+#
+# Send an event
+#
+class CombinedReceiveEventTestCase(unittest.TestCase):
+
+    def sendEvent(self, broker, topic):
+        trans = events.EventTransmitter(broker, topic)
+        
+        root = PropertySet()
+        root.set("TOPIC",topic)
+        root.set("myname","myname")
+        root.set("STATUS", "my special status")
+        root.set("RUNID","srptestrun")
+        
+        event = events.Event("srptestrun", root)
+    
+        # wait a short time so we can see the difference between the time 
+        # the event is created and the time it is published
+        time.sleep(2)
+    
+        # ok...now publish it
+        trans.publishEvent(event)
+
+    def testCombinedReceive(self):
+        # Test sending on two topics, and receiving bother on a third.
+        #
+        broker = "lsst8.ncsa.illinois.edu"
```

Comment:
This should work everywhere.  This is a long standing issue about what to do about the activemq broker on offsite systems, and a probably with nearly all of these tests in general.